docs/FAQ.md

```diff
@@ -183,10 +183,12 @@ For the equivalent of `git commit --amend -p`/`hg amend -i`, use `jj squash -i`.
 
 ### Is there something like `git rebase --interactive` or `hg histedit`?
 
-Not yet, you can check [this issue] for updates.
+It is often sufficient to use some form of `jj rebase` with `-A/-B`. For
+example, if you have a linear chain of revisions `A` through `E` and want to
+move `C` and `E` before `B`, use `jj rebase -r 'C|E' -B B`. See
```

Comment:
I think this example is too confusing at this length. We could have an example with moving just one commit before another and then say "You can also move multiple commits at once; see `jj help rebase` for more examples".

Or, we could have a simple example and a more detailed explanation of a more complicated example later. In particular, I'd at least illustrate the initial sequence of commits `A -> B -> C -> D -> E`, show the result, and note that `rebase` took notice that E is a descendant of C.

Reply:
Makes sense. I simplified it to just 3 commits and moving one of them.